src/main/java/seedu/multitasky/storage/StorageDateConverter.java

```diff
@@ -0,0 +1,60 @@
+package seedu.multitasky.storage;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+//@@author A0132788U
+/**
+ * Class that implements methods to convert Date to String to enable Storage to store dates in a human-editable form in
+ * the XML file.
+ * Parses String back to Date and returns to Model.
+ */
+public class StorageDateConverter {
+
+    /** Formatter to parse date into a human-editable string to store in the XML file */
+    private DateFormat formatter = new SimpleDateFormat("dd/MM/yy HH:mm");
```

Comment:
hmmm, now that the parser and UI display is sticking to MM/dd/yy HH:mm format, maybe we should do the same for storage for consistency?